### ISSUE TITLE
bug: iPad crashes after set duration- sometime around the "grand experiment" scene

### ISSUE THREAD

**Issue by lindamood** [nprapps/rockymountain#149]

**Comment by TylerFisher** [comment]
@brmayes did you see this happen in your testing?

**Comment by brmayes** [comment]
Yes! It reloaded itself because it said there was a problem with the webpage. 

Also interesting: as this was approaching, the transitions between the scenes got longer and more noticeably glitchy.

**Comment by TylerFisher** [comment]
We tested on @ixdavid's iPad Pro and everything worked fine.

This seems like a memory issue, or maybe graphics processing, with older iPads. Beyond further optimization of images, there isn't much I can do about that.

It doesn't seem like a loading issue, it's more about the iPad freaking out once it's had to display a number of scenes. The bug was produceable in both letting the story run and clicking around to a bunch of MoZ cards at any pace.
